### ISSUE TITLE
python stopped working using pytorch script

### ISSUE THREAD

**Issue by yana25** [peterjc123/pytorch-scripts#17]
Hi,

I don't know if my problem belong here.
I'm using pytorch scripts on windows and i'm having issues converting my model to fp16 using ".half()" on the entire model and converting BatchNorm layers to .float()
When I run my code it crashes and give me an error "Python stopped working" without showing any exception.
I searched in the Event Viewer the reason why and got the following:
`
Faulting application name: python.exe, version: 3.6.4150.1013, time stamp: 0x5a5e439a
Faulting module name: _C.cp36-win_amd64.pyd, version: 0.0.0.0, time stamp: 0x5a91cbee
Exception code: 0xc0000005
Fault offset: 0x000000000078cd95
Faulting process id: 0x2c78
Faulting application start time: 0x01d3b5ee018887d6
`

**Comment by peterjc123** [comment]
@yana25 Sorry, half tensors are not supported yet. Would you please use `.float()` instead of `.half()`?

**Comment by yana25** [comment]
@peterjc123 will it convert my model to 16fp? or it still be in fp32?

**Comment by peterjc123** [comment]
FP16 is not supported on Windows. If you use `.float`, then it will be FP32.

**Comment by yana25** [comment]
ok, thanks!
